### ISSUE TITLE
How about exporting some query option type

### ISSUE THREAD

**Issue by hyoban** [Crossbell-Box/crossbell.js#84]
Like `NoteQueryOptions`

**Comment by Songkeys** [comment]
Absolutely! We can definitely make these adjustments. Currently, some functions have varying parameter types, with some separated and some not. It would be great to separate and export them accordingly.

Regarding the function parameter style:

```ts
// Some are like this:
getNote(characterId: number, noteId: number)

// While some are like this:
getNotes(options: NoteQueryOptions)
```

I believe sticking to the second format consistently is the way to go. It's a significant change that we should implement in v2.

```ts
// Example:
type GetNoteQueryOptions = {
  characterId: number
  noteId: number
}
getNote(options: GetNoteQueryOptions)
```

But for the current v1, we can do the exporting anyway.
